cli/operator/keys/import.go

```diff
@@ -0,0 +1,109 @@
+package keys
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/Layr-Labs/eigensdk-go/crypto/bls"
+	"github.com/NethermindEth/eigenlayer/cli/prompter"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/spf13/cobra"
+)
+
+func ImportCmd(p prompter.Prompter) *cobra.Command {
+	var (
+		keyType  string
+		insecure bool
+		help     bool
+	)
+
+	cmd := cobra.Command{
+		Use:   "import --key-type <key-type> [flags] <keyname> <private-key>",
+		Short: "Used to import existing keys in local keystore",
+		Long: `
+Used to import ecdsa and bls key in local keystore
+
+keyname (required) - This will be name of the imported key file. It will be saved as <keyname>.ecdsa.key.json or <keyname>.bls.key.json
```

Comment:
```suggestion
keyname (required) - This will be the name of the imported key file. It will be saved as <keyname>.ecdsa.key.json or <keyname>.bls.key.json
```